memory_addr/src/lib.rs

```diff
@@ -77,72 +77,20 @@ pub const fn is_aligned_4k(addr: usize) -> bool {
 
 #[cfg(test)]
 mod tests {
-    use crate::{va, va_range, VirtAddrRange};
+    use super::*;
```

Comment:
I think it's a goot habit to avoid the using of `use xxx::*`.

Reply:
relax, it's `mod tests`